packages/block-library/src/navigation-menu/index.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Server-side rendering of the `core/navigation-menu` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/navigation-menu` block on server.
+ *
+ * @param array $attributes The block attributes.
+ * @param array $content The saved content.
+ * @param array $block The parsed block.
+ *
+ * @return string Returns the post content with the legacy widget added.
+ */
+function render_block_navigation_menu( $attributes, $content, $block ) {
+	return prepare_navigation( $block );
+}
+
+/**
+ * Walks the inner block structure and retunrs an HTML list of it.
+ *
+ * @param array $block The block.
+ *
+ * @return string Returns  an HTML list from innerBlocks.
+ */
+function prepare_navigation( $block ) {
```

Comment:
`prepare_navigation` doesn't really describe what this does. We're not _preparing_ anything. How about something like `build_navigation_menu_html`?

Reply:
you're right, leftover from when using the Walker